src/index.js

```diff
@@ -71,7 +71,8 @@ app.get('/:id', function(req, res) {
 
   res.writeHead(200, {
     'Content-Type': 'application/zip',
-    'Content-disposition': 'attachment; filename="' + filename + '"'
+    'Content-disposition': 'attachment; filename="' + filename + '"',
+    'Content-Lenght':filename.size
```

Comment:
How could I misspell length ?
I will find a way. But is the zipping process finished when the response is sent ?